### ISSUE TITLE
Arena-like UX (animations/sounds/etc)

### ISSUE THREAD

**Issue by Victor239** [magefree/mage#8301]
Arena has some awful flaws like it's grind to get rares, addictive features like daily wins and lack of fun features like commander.

But what it does well is UX. Beginners to Magic can learn the game quickly via MtG: Arena since it has many cues like varying sound effects depending on power of creature attacking, unique particle effects for different types of playable exiled cards, sound effects for various cards played, etc.

These all make for a more entertaining experience and makes it easier to tell at a glance or listen what is going on, without having to rely on a text log telling you what happened in a turn.

These examples and others would be superb for attracting people put off by the existing UX such as myself. I'd be willing to contribute to a bounty towards these kinds of features if there's interest in it on the dev side.

**Comment by JayDi85** [comment]
Yes, it's possible in theory. XMage is server side project, so you can use any third party client to connect and play games with other players.

**Comment by jeffwadsworth** [comment]
Personally, that bells and whistles arcade approach is why I left Arena.  Mage works quite well as-is.

**Comment by Victor239** [comment]
@jeffwadsworth additional sound effects or animations should be able to be turned off still for users like yourself, but having the option available would additionally suit users like me.
